doc/nrf/gs_programming.rst

```diff
@@ -124,21 +123,32 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
 
    To build and program an application:
 
-      a. Select your project in the Project Explorer.
-      #. From the menu, select :guilabel:`Build` > :guilabel:`Build Solution`.
-      #. When the build completes, you can program the application to a connected development kit:
+   a. Select your project in the Project Explorer.
+      The project name displays in bold when it is selected.
+   #. From the menu, select :guilabel:`Build` > :guilabel:`Build Solution`.
+   #. When the build completes, you can program the application to a connected development kit:
 
-         * For a single-image application, select :guilabel:`Target` > :guilabel:`Download zephyr/zephyr.elf`.
-         * For a multi-image application, select :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
+      * For a single-image application, select :guilabel:`Target` > :guilabel:`Download zephyr/zephyr.elf`.
+      * For a multi-image application, depending on your build target:
 
-      .. warning::
-	   For nRF53 family SoC, programming :file:`merged.hex` file updates only the application core.
-	   If you need to update also the network core, you must follow additional steps described in :ref:`ug_nrf5340`.
+        * If you are programming a SoC from the nRF53 Series and you also need to update the network core, you must add the network core project in |SES| and complete the additional steps, as described in the :ref:`ug_nrf5340_ses_multi_image` section of :ref:`ug_nrf5340`.
+          This is because programming the :file:`merged.hex` file at this stage updates only the application core.
+        * If you are not programming an nRF53 Series SoC or you do not need to update the network core, select :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
 
-      .. note::
+   .. note::
 	   Alternatively, choose the :guilabel:`Build and Debug` option.
-	   :guilabel:`Build and Debug` will build the application and program it when
-	   the build completes.
+	   :guilabel:`Build and Debug` will build the application and program it when the build completes.
+
+   Regardless of the chosen option, ignore any project out-of-date warnings by clicking :guilabel:`No` when they appear.
+
+   .. figure:: images/ses_nrf5340_netcore_download.png
+      :alt: Ignore any 'Project out-of-date' warnings
+
+      Ignore any 'Project out-of-date' warnings
+
+   .. warning::
+      If you click :guilabel:`Yes` and disable the option to show the dialog again, you will enter a loop because of a "no input files" error.
+      To restore the default settings, go to :guilabel:`Tools` > :guilabel:`Options` > :guilabel:`Building` and set :guilabel:`Confirm Automatically Build Before Debug` to ``Yes``.
```

Comment:
```suggestion
   .. note::
      If you click :guilabel:`Yes` and disable the option to show the dialog again, you will enter a loop because of a "no input files" error.
      To restore the default settings, select :guilabel:`Tools` > :guilabel:`Options` > :guilabel:`Building` and set :guilabel:`Confirm Automatically Build Before Debug` to ``Yes``.
```